src/main/java/org/gridsuite/filter/server/dto/AbstractInjectionFilter.java

```diff
@@ -28,7 +29,10 @@ public abstract class AbstractInjectionFilter extends AbstractEquipmentFilterFor
     String substationName;
 
     @Schema(description = "Countries")
-    private SortedSet<String> countries;
+    private Set<String> countries;
```

Comment:
Because SortedSet is not the right interface here.
I made experiment around that and forgot to revert on some places. I revert.
"Sorted" is not the right interface, because what matters most in the order aspect is the order user (or front) gave and has to be kept. I came aware of this with the properties where order changes appear more blattant.

Reply:
Reverted